frontend/src/metabase/components/ErrorPages/DownloadDiagnosticModal.tsx

```diff
@@ -0,0 +1,65 @@
+import { t } from "ttag";
+
+import { Form, FormProvider, FormSubmitButton } from "metabase/forms";
+import { Modal, Text } from "metabase/ui";
+
+import { DiagnosticCheckboxes } from "./DiagnosticCheckboxes";
+import type { ErrorPayload } from "./types";
+
+interface DownloadDiagnosticModalProps {
+  errorInfo: ErrorPayload;
+  onClose: () => void;
+  onSubmit: (values: any) => void;
```

Comment:
Would it be possible to avoid using `any` and a bit generic `Record<string, boolean>` type for `hiddenValues` below?

Reply:
I noticed it in other files as well, adding precise types would help a lot